commit 22e0f06767fe
Author: Valentin Gosu <valentin.gosu@gmail.com>
Date:   Tue Dec 5 02:29:41 2017 +0100

    Bug 1422518 - Use LOAD_FLAGS_DISALLOW_INHERIT_PRINCIPAL for meta refresh r=bz a=gchang
    MozReview-Commit-ID: H3hnO3NmHYA
    
    --HG--
    extra : rebase_source : b9f7cbccdd2f2723e6d66d89f9ac529d462d75dd
    extra : source : 6f5aec4cde6f874921eaa1b0b9d7e9d4c7feabe0
---
 docshell/base/nsDocShell.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git docshell/base/nsDocShell.cpp docshell/base/nsDocShell.cpp
index 83cb87dd8255..31d414d1b57d 100644
--- docshell/base/nsDocShell.cpp
+++ docshell/base/nsDocShell.cpp
@@ -6976,7 +6976,7 @@ nsDocShell::ForceRefreshURI(nsIURI* aURI, nsIPrincipal* aPrincipal, int32_t aDel
    * LoadURI(...) will cancel all refresh timers... This causes the
    * Timer and its refreshData instance to be released...
    */
-  LoadURI(aURI, loadInfo, nsIWebNavigation::LOAD_FLAGS_NONE, true);
+  LoadURI(aURI, loadInfo, nsIWebNavigation::LOAD_FLAGS_DISALLOW_INHERIT_PRINCIPAL, true);
 
   return NS_OK;
 }
